src/test/java/api/support/fakes/FakeOkapi.java

```diff
@@ -291,6 +291,13 @@ public void start(Future<Void> startFuture) throws IOException {
       .create()
       .register(router);
 
+    new FakeStorageModuleBuilder()
+      .withCollectionPropertyName("expiredSessions")
+      .withRootPath("/patron-action-session-storage/expired-session-patron-ids")
+      .withQueryParameters("action_type", "session_inactivity_time")
```

Comment:
Are these query parameters required to use this endpoint?

Reply:
session_inactivity_time is required parameter, action_type is not required but we use it to identify the type of action, in our case it is Check-out